pkg/cmd/cli/kubeflowmodelregistry/kfmr.go

```diff
@@ -52,16 +50,12 @@ func LoopOverKFMR(owner, lifecycle string, ids []string, writer io.Writer, forma
 			if err != nil {
 				klog.Errorf("%s", err.Error())
 				klog.Flush()
-				return nil, nil, err
```

Comment:
Moving the call the `CallBackstagePrinters` out of `LoopOverKFMR` and then call it looping over the results from `LoopOverKFMR`, but using a new `Writer` each time, was the key bit in not including multiple model catalogs in one JSON entry .... with your catalog-info.YAML that data structure / indexing was as such that we could use one `Writer`